assets/src/modules/Digitizing.js

```diff
@@ -472,19 +472,23 @@ export default class Digitizing {
     import(file){
         const reader = new FileReader();
 
+        // get extension file
+        var fileExtension = file.name.split('.').pop();
+
         reader.onload = (() => {
             return (e) => {
                 const fileContent = e.target.result;
                 let OL6features;
 
                 // Handle GeoJSON, GPX or KML strings
                 try {
-                    if (fileContent[0] === '{') {
+                    // Check extension to the feature type
+                    if (fileExtension === 'geojson') {
```

Comment:
What if the extenion is json ?

Reply:
I can make `if (fileExtension === 'geojson' || fileExtension === 'json')` ?

Reply:
@drillinP I'll do it

Reply:
Done